content/ember/v6/deprecate-ember-mixin.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Deprecating Ember.Mixin
+until: 7.0.0
+since: 6.5.0
+---
+
+`Ember.Mixin` is deprecated. This is part of the legacy `@ember/object` API and is not fully compatible with native classes. Instead of using mixins, you should refactor your code to use class-based patterns.
```

Comment:
```suggestion
`Ember.Mixin` is deprecated. This is part of the legacy `@ember/object` API and is not compatible with native classes. Instead of using mixins, you should refactor your code to use class-based patterns.
```